src/server/controllers/items.js

```diff
@@ -0,0 +1,25 @@
+const Item = require('../models/item.model');
+
+// Get all the items
+const getAllItems = function(req, resp) {
+  switch (req.params.type) {
+    case 'ordered':
+      Item.find({ isOrdered: true }, (err, items) =>
```

Comment:
pull out this to some variable like: findConditions and call the Item.find(findConditions...) only once